components/PlantCard/PlantCard.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import styles from './PlantCardStyles.module.css';
+
+function format(start: String, end: String) {
```

Comment:
are we using this function anywhere? if not, we can remove it?

Reply:
oh shoot that was a remnant of an older time will remove now